app/weeklyPicks/page.tsx

```diff
@@ -31,41 +32,37 @@ const FormSchema = z.object({
 
 export default function WeeklyPickForm() {
   const [NFLTeams, setNFLTeams] = useState<Models.Document[]>([]);
-  const [weeklyPicks, setWeeklyPicks] = useState<Models.Document>();
+  const [userPick, setUserPick] = useState<string | null>(null);
+  const [userId, setUserId] = useState<string | null>(null);
+  const [allPicks, setAllPicks] = useState<string | null>(null);
 
-  // TODO: Combining both use Effects into one call
   useEffect(() => {
-    async function fetchNFLTeams() {
-      const data = await getNFLTeams();
-      if (data) {
-        const response = data.documents;
-        setNFLTeams(response);
-      }
-    }
+    async function fetchWeeklyPicks() {
+      try {
+        const allPicks = await getAllWeeklyPicks();
 
-    fetchNFLTeams();
-  }, []);
+        const data: any = await getNFLTeams();
 
-  useEffect(() => {
-    async function fetchWeeklyPicks() {
-      const user = await account.get();
+        const response = data.documents;
+        setNFLTeams(response);
 
-      const userPicks = await getUserWeeklyPick({
-        userId: user.$id,
-        weekNumber: '6622c75658b8df4c4612',
-      });
+        const user = await account.get();
 
-      const nflTeams = await getNFLTeams();
+        setUserId(user.$id);
 
-      if (nflTeams && userPicks) {
-        const userResponse = userPicks;
-        const nflResponse = nflTeams.documents;
+        const userPickedTeam = await getUserWeeklyPick({
+          userId: user.$id,
+          weekNumber: '6622c75658b8df4c4612',
+        });
 
-        const findTeam = nflResponse.find(
-          (team) => team.$id === userResponse[user.$id].team,
-        );
+        if (userPickedTeam) {
+          setUserPick(userPickedTeam);
+        }
 
-        setWeeklyPicks(findTeam);
+        if (allPicks?.documents[0].userResults != '')
+          setAllPicks(allPicks?.documents[0].userResults);
```

Comment:
Let's not use any shorthand code. It's much easier for all skill levels to read the code when it's not short handed. 
```suggestion
        if (allPicks?.documents[0].userResults != '') {
          setAllPicks(allPicks?.documents[0].userResults);
        }
```